packages/luna/tailwind.config.js

```diff
@@ -261,9 +261,14 @@ export default {
         4: "4",
         5: "5",
       },
+      lineHeight: {
+        "3-half": "14.4px",
+        "4-half": "19.2px",
+      },
```

Comment:
Discuss with Ziad that this line heights are not part of the design system.

Reply:
Ziad just updated the designs and now he is using the default line-hieght for each element.